src/Cms/System.php

```diff
@@ -322,12 +322,11 @@ public function license()
 
 		// only return the actual license key if the
 		// current user has appropriate permissions
-		$user = $this->app->user();
-		if ($user && $user->isAdmin() === true) {
+		if ($this->app->user()?->isAdmin() === true) {
```

Comment:
Here I like it on the other hand because it really makes the code a lot shorter and increases the overview.